Resources/input/fsh/ruleset.fsh

```diff
@@ -0,0 +1,19 @@
+RuleSet: Meta
```

Comment:
Etwas offtopic aber könnte man "Standard" Rulesets irgendwo zentral mal hosten? Habe etwas Varianz zwischen den Projekten gesehen. Einige geben bei Meta z.B. noch ^url mit an. Könnte man gesondert noch mal betrachten.

Reply:
Das würde ich nicht zentralisieren. Lass es uns in jedes Projekt einbringen, das wir anfassen und die Verantwortung für tragen.